src/setup.rs

```diff
@@ -118,10 +118,12 @@ async fn run_server_with_runtimes<T>(
 
     // Config limiter
     let limiter = Limiter::new(config.limiter.clone());
+    let config_content = serde_json::to_string(&config).expect("Fail to serialize config");
```

Comment:
- https://docs.rs/toml/latest/toml/ser/fn.to_string.html

Toml format is consistent with our config.

Reply:
```
unwrap_or_else(|_| {print_err("");  "Unable to serialize config" })
```